RELEASE/scripts/autoscend.ash

```diff
@@ -2407,24 +2527,34 @@ boolean LX_attemptPowerLevel()
 {
 	set_property("auto_powerLevelAdvCount", get_property("auto_powerLevelAdvCount").to_int() + 1);
 	set_property("auto_powerLevelLastAttempted", my_turncount());
-
+	set_property("auto_powerLevelLastLevel", my_level());
+	set_property("auto_newbieOverride", true);
+	
 	handleFamiliar("stat");
 	addToMaximize("100 exp");
-
-	if(LX_freeCombats(true))
+	
+	auto_log_warning("I need to powerlevel", "red");
+	int delay = get_property("auto_powerLevelTimer").to_int();
```

Comment:
neat, I didn't know about this property. Can you add it to the GUI while you're at it?

Reply:
Good idea. will do... tomorrow, I am falling asleep now.

Reply:
change made